BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -1148,6 +1147,11 @@ public void Start()
                     {
                         if (mtf.Current == null) continue;
                         KSPParticleEmitter kpe = mtf.Current.GetComponent<KSPParticleEmitter>();
+                        if (kpe == null)
+                        {
+                            Debug.Log("[BDArmory.ModuleWeapon] MuzzleFX transform missing KSPParticleEmitter component. Please fix your model");
```

Comment:
These should be upgraded to `LogWarning` or `LogError` so they're easier to spot in the log.